docs/prompt_behavior.md

```diff
@@ -0,0 +1,16 @@
+| MSAL    | `LOGIN`           | Sends query param `prompt=login` to `/authorize` endpoint. The user will _always_ be prompted for credentials by the service.                                                                                                                              | `login`                                                                                          | Yes                  | Same as embedded flow.                                                                                                                                                                                                                                                                                                 | `login`                                                   |
+| MSAL    | `CONSENT`         | Sends query param `prompt=consent` to `/authorize` endpoint. The user will be prompted to grant consent, even if it was previously granted before. `// TODO what happens for pre-authz apps?`                                                              | `consent`                                                                                        | Yes                  | BUG: User is not shown prompt for consent. <br/>Expected: User shown consent page                                                                                                                                                                                                                                      | `consent`                                                 |
+| MSAL    | `WHEN_REQUIRED`   | If `login_hint` is supplied, no `prompt` query parameter will be sent to the `/authorize` endpoint. The user _may_ be preompted to login or grant consent as required by the service/application. If no `login_hint` is supplied, `prompt=select_account`. | If `login_hint` is supplied, unsent <br/> If no `login_hint` is supplied, sends `select_account` | N/A                  | Same as embedded flow                                                                                                                                                                                                                                                                                                  | (unsent - account selection via `AccountChooserActivity`) |
+
+>Note: `CONSENT` not implemented inside of broker with MSAL
```

Comment:
File a bug

Reply:
Make table reflect _desired_ behavior once bug is filed

Reply:
Bug filed here:
https://github.com/AzureAD/ad-accounts-for-android/issues/1294

Contains link to VSO PBI